ubiquitous-journey/values-day2ops.yaml

```diff
@@ -50,3 +50,12 @@ applications:
     values:
       namespace: *ops_ns
       prune_type: deployments
+  # Network Policies for Emulating Multitenancy
+  - name: network-policy
+    enabled: true
```

Comment:
I'm not sure i would enable this Chart by default. NetwrokPolicies are very application and cluster centric depending on the use cases. Thoughts ?

Reply:
You're right! This is just for the given namespace yet could be break/override stuff, especially if the cluster has already some policies in-place.